src/App.js

```diff
@@ -1,18 +1,49 @@
 import * as React from 'react';
 
-function getTitle(title){
-  return title;
-}
+const list = [
```

Comment:
You were supposed to create an array with todos and store it in a variable named todoList.

Reply:
Then you were supposed to create at least 3 objects with the following properties: id: unique identifier (e.g., 1, 2, 3) title: summary of the todo item (e.g., 'Complete assignment')